test_runner/regress/test_broken_timeline.py

```diff
@@ -10,7 +10,7 @@
 
 # Test restarting page server, while safekeeper and compute node keep
 # running.
-def test_broken_timeline(neon_env_builder: NeonEnvBuilder):
+def test_broken_timeline_zzz(neon_env_builder: NeonEnvBuilder):
```

Comment:
?

Reply:
Oh, that's maybe due to be not knowing pytest enough, but when I run things with `poetry run pytest -sk "test_broken_timeline"` , all tests that start with test_broken_timeline are run, which could be long.
As a simple quick workaround, I've renamed the test, but forgot to slap a TODO on it.

Reply:
Heh, I do the same, let me know if you come up with a better solution :-)